python/lsst/obs/cfht/cfhtIsrTask.py

```diff
@@ -105,10 +108,12 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
                 # (visit : 7xxxxx) where RDNOISEA/B = 65535
                 if rdnB > 60000.0:
                     rdnB = metadata.getScalar("RDNOISE")
-                amp.setReadNoise(rdnB)
+                tempAmp.setReadNoise(rdnB)
             else:
                 raise ValueError("Unexpected amplifier name : %s"%(amp.getName()))
+            tempCcd.append(tempAmp)
 
+        ccd = tempCcd.finish()
```

Comment:
You go through all this work, but don't actually do anything with the variable you create. Do you mean to do ccdExposure.setDetector or something here?